plugins/cybereason/icon_cybereason/util/api.py

```diff
@@ -219,7 +224,16 @@ def parse_file_filter(file_filter: str):
         return [{"fieldName": file_name, "operator": operator, "values": value[file_filter[0]]}]
 
     @staticmethod
-    def generate_sensor_filter(identifier: str) -> dict:
+    def which_filter(identifier: str) -> str:
+        if validators.ipv4(identifier):
+            return "internalIpAddress"
+        elif re.match(r"^(-?\d{9,10}\.-?\d{19})$", identifier):
+            return "guid"
```

Comment:
Would the validators.uuid check work for this instead of he re.match?

Reply:
it looks like that approach didnt work. the guid they have is not of the typical uuid format.